### ISSUE TITLE
Enable voice call

### ISSUE THREAD

**Issue by karlsequeira** [okta/terraform-provider-okta#247]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description
I am not able to setup voice call on the password policy

### New or Affected Resource(s)

- https://registry.terraform.io/providers/oktadeveloper/okta/latest/docs/resources/policy_password

### Potential Terraform Configuration

call_recovery

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

- #0000

**Comment by bogdanprodan-okta** [comment]
Hi @karlsequeira! Thanks for submitting this issue! I'll take a look at it more closely and get back to you.

**Comment by bogdanprodan-okta** [comment]
Hi @karlsequeira! I've created PR that adds the ability to set up a voice call on the password policy. It will be available in v3.7.3 soon!
